### ISSUE TITLE
"No suspicious code found". although there are errors

### ISSUE THREAD

**Issue by reyem** [omayevskiy/sonar-intellij-plugin#22]
Hi,
I used the sonar connector (http://plugins.jetbrains.com/plugin/7238) plugin to sync rules from sonar server to the intellij inspection rules. When I just let these rules run, I correctly see the number of warnings in the top right corner of the editor, but intelliJ still reports "No suspicious code found". With other inspections it works... but no error message shows up. See screenshot.

Intellij 12.1.6 on Mac OS 10.9

thank you for help
![bildschirmfoto 2013-11-15 um 14 53 44](https://f.cloud.github.com/assets/5948347/1550186/5ce51b2c-4dfd-11e3-9410-97072ad2516d.png)

**Comment by engrun** [comment]
I have the same problem
Same setup - Intellij 12.1.6 on Mac OS 10.9

**Comment by engrun** [comment]
Upgraded to latest version today (1.4.x) - still a problem

**Comment by reyem** [comment]
Agree. Still not working with plugin version 1.4. Sonar server version 3.6.

**Comment by reyem** [comment]
It does not work for a single file. But if I configure the module in the module settings, the analysis of the whole module actually works. After I performed this, also the analysis of a single file works.
However, the results I see are actually the server side version of the inspection results. Therefore I'm not sure if this ok. I was expecting that the rules are synced to the client and I can perform the same checks locally as the server does, and therefore would be able to fix the issues before commit. But my expectation might be wrong...

**Comment by engrun** [comment]
@reyem I agree. This is also my expectation. 
I am currently using QAPlug to perform the analysis, but since Sonar is migrating to Squid/SSLR rules, and deprecating PMD/Checkstyle rules, I'm looking for a solution to support squid. (qaplug is not currently supporting this)

So the point would be to analyze locally, and fix issues before commiting and pushing

**Comment by jblack10101** [comment]
I'm running into this issue with Intellij 13.1.5
